homeassistant/components/abode/binary_sensor.py

```diff
@@ -61,6 +62,16 @@ def device_class(self):
 class AbodeQuickActionBinarySensor(AbodeAutomation, BinarySensorDevice):
     """A binary sensor implementation for Abode quick action automations."""
 
+    def __init__(self, data, automation, event):
```

Comment:
We don't need this init since the parameters are the same here and in the parent init method.